modules/claims_api/app/swagger/claims_api/v1/swagger.json

```diff
@@ -496,7 +496,7 @@
               "application/json": {
```

Comment:
Doesn't look like anything really changed in this (other than obscure metadata).
Anytime you run the rswag logic, this file is going to have metadata modifications like this.  I've just been discarding changes like this so we don't end up pushing changes that aren't really "changes". (if that makes sense)  
Just something to consider in the future.

Reply:
that's a good point. I'll make sure to only update v2 for the other changes below